src/components/Layout/Header.tsx

```diff
@@ -11,7 +11,7 @@ import Menu from '@material-ui/core/Menu'
 import MenuItem from '@material-ui/core/MenuItem'
 import Divider from '@material-ui/core/Divider'
 import ListItemIcon from '@material-ui/core/ListItemIcon'
-import ocLogo from '../../assets/images/four51-badge.svg'
+import ocLogo from '../../assets/images/four51-badge--orange.svg'
```

Comment:
Could we get rid of this change - since the header will be totally re-done in the next PR?

Reply:
shouldn't hurt to rename the file, is it conflicting on your new nav or something?

Reply:
It will, because I have the whole logo in there, but I guess we could be good with just the favicon.  If yes, I'll merge.

Reply:
oh i see, yeah I suppose we should have got your sidenav in there before worrying about renaming assets. could you just update your logo with the name of the new full logo? `src/assets/images/four51-badge--{ORANGE/WHITE/FULL-COLLOR}.svg`

Reply:
Can do, so I want `four51-logo--full-color` correct?

Reply:
yep `src/assets/images/four51-logo--full-color.svg` is golden! That's the orange/teal one.